src/main/java/ru/skypro/homework/controller/AdController.java

```diff
@@ -17,136 +17,56 @@
-            )
-    }
-    )
-    @GetMapping("/ads/{id}")
+    @GetMapping("/{id}")
     public ResponseEntity<ExtendedAd> adInfoById(@PathVariable Integer id) {
         return ResponseEntity.ok().build();
     }
 
     //
 
-    @ApiResponses(value = {
-            @ApiResponse(
-                    responseCode = "204", description = "запрос выполнен, нет возвращаемогосодержимого;"
-            ),
-            @ApiResponse(
-                    responseCode = "401", description = "отсутствует авторизация;"
-            ),
-            @ApiResponse(
-                    responseCode = "403", description = "операция запрещена;"
-            ),
-            @ApiResponse(
-                    responseCode = "404", description = "объявление не найдено;"
-            )
-    }
-    )
-    @DeleteMapping("/ads/{id}")
+    @DeleteMapping("/{id}")
     public ResponseEntity<?> deleteAdById(@PathVariable Integer id) {
         return ResponseEntity.ok().build();
     }
 
     //
 
-    @ApiResponses(value = {
-            @ApiResponse(
-                    responseCode = "200", description = "запрос выполнен;"
-            ),
-            @ApiResponse(
-                    responseCode = "401", description = "отсутствует авторизация;"
-            ),
-            @ApiResponse(
-                    responseCode = "403", description = "операция запрещена;"
-            ),
-            @ApiResponse(
-                    responseCode = "404", description = "объявление не найдено;"
-            )
-    }
-    )
-    @PatchMapping("/ads/{id}")
+    @PatchMapping("/{id}")
     public ResponseEntity<?> patchAdById(@PathVariable Integer id, @RequestBody CreateOrUpdateAd createOrUpdateAd) {
         return ResponseEntity.ok().build();
     }
 
     //
 
-    @ApiResponses(value = {
-            @ApiResponse(
-                    responseCode = "200", description = "запрос выполнен;"
-            ),
-            @ApiResponse(
-                    responseCode = "401", description = "отсутствует авторизация;"
-            )
-    }
-    )
-    @GetMapping("/ads/me")
+    @GetMapping("/me")
     public ResponseEntity<?> showAuthorizedUserAd() {
         return ResponseEntity.ok().build();
     }
 
     //
 
-    @ApiResponses(value = {
-            @ApiResponse(
-                    responseCode = "200", description = "запрос выполнен;"
-            ),
-            @ApiResponse(
-                    responseCode = "401", description = "отсутствует авторизация;"
-            ),
-            @ApiResponse(
-                    responseCode = "403", description = "операция запрещена;"
-            ),
-            @ApiResponse(
-                    responseCode = "404", description = "объявление не найдено;"
-            )
-    }
-    )
-    @PatchMapping("/ads/{id}/image")
-    public ResponseEntity<?> patchAdPictureById(@PathVariable Integer id, @RequestBody String linkedPicture) {
+    @PatchMapping("/{id}/image")
```

Comment:
```suggestion
   @PatchMapping(value = "/{id}/image", consumes = MediaType.MULTIPART_FORM_DATA_VALUE)
```
вот так должно быть, иначе не будет работать